WATOLINK/bci-gaming
# Grace - Death sequence

## ClickUp ticket link

[Game - Death Sequence](https://app.clickup.com/t/860q9n3j4)



## Implementation description
Game over screen with restart level, return to main menu, and quit game buttons. The game over screen is setup with a canvas image, and all graphic elements including text font, button style, etc are from Unity.   
Note: This ticket is concurrent with the Game - Health and Game - Take Damage tickets. Please review this ticket first so I could move onto the other tickets.



## Steps to test
1. Click play and make sure that the scene loads
2. Restart button: Onclick the scene would get reloaded. Currently you couldn't test this since the health bar & taking damage tickets need to be finished before this could be tested. For now, just confirm that no errors occur onclick.
3. Menu button: Onclick the player should be directed to the menu scene.
4. Quit button: Quit game.



## What should reviewers focus on?



## Checklist
